src/components/editor/document-bar/menus/export/markdown.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { Dropdown } from 'react-bootstrap'
+import { ForkAwesomeIcon } from '../../../../common/fork-awesome/fork-awesome-icon'
+
+export interface MarkdownExportProps {
+  title: string
+  noteContent: string
+}
+
+export const MarkdownExport: React.FC<MarkdownExportProps> = ({ title, noteContent }) => {
+  const downloadMarkdown = () => {
+    const element = document.createElement('a')
+    const file = new Blob([noteContent], { type: 'text/markdown' })
+    element.href = URL.createObjectURL(file)
+    element.download = `${title}.md`
+    document.body.appendChild(element) // Required for this to work in FireFox
+    element.click()
```

Comment:
The `element` should be removed again after download, otherwise the DOM will be cluttered when clicking the button multiple times. For reference have a look at [history-utils](https://github.com/codimd/react-client/blob/HEAD/src/components/history-page/utils.ts#L137).

![image](https://user-images.githubusercontent.com/52606896/94947876-a9072280-04de-11eb-99d4-2dfe250be257.png)

Reply:
I think I should use the helper function from #628